Gemfile

```diff
@@ -26,7 +30,7 @@ gem 'uglifier', '>= 1.3.0'
 # jQuery & plugins
 gem 'jquery-turbolinks'
 gem 'jquery-rails'
-gem 'jquery-ui-rails'
+gem 'jquery-ui-rails', '~> 5.0'
```

Comment:
I think it's not needed

Reply:
The version suffix, `'~ 5.0'`? Or the Gem itself?

Reply:
Version suffix